qemu/tests/cfg/hugepage_specify_node.cfg

```diff
@@ -0,0 +1,11 @@
+- hugepage_specify_node:
+    type = hugepage_specify_node
+    only ppc64 ppc64le
+    virt_test_type = qemu
+    kill_vm = yes
+    setup_hugepages = yes
+    hugepage_size = 4096
+    target_hugepages = 2048
```

Comment:
If host uses 1G hugeapge, this parameter is incorrent.

Reply:
correct